packages/react-charting/src/utilities/utilities.ts

```diff
@@ -1087,7 +1143,36 @@ export function getDomainNRangeValues(
     switch (chartType) {
       case ChartTypes.AreaChart:
       case ChartTypes.LineChart:
```

Comment:
>ChartTypes [](http://example.com/codeflow?start=11&length=10)

this can be a single group of cases for area, line, vbc, stacked vbc

Reply:
Done